src/docs/cookbook/testing/widget/scrolling.md

```diff
@@ -159,4 +159,4 @@ flutter test test/widget_test.dart
 [`WidgetTester`]: {{site.api}}/flutter/flutter_test/WidgetTester-class.html
 [`ListView.builder`]: {{site.api}}/flutter/widgets/ListView/ListView.builder.html
 [`scrollUntilVisible()`]: {{site.api}}/flutter/flutter_test/WidgetController/scrollUntilVisible.html
-[Work with long lists]: /docs/cookbook/lists/long-lists
\ No newline at end of file
+[Work with long lists]: /docs/cookbook/lists/long-lists
```

Comment:
I'm not sure why this line's changed - I didn't touch it?

Reply:
I'm guessing end of line white space changed